doc/changelog.rst

```diff
@@ -10,6 +10,9 @@ PyMongo 4.9 brings a number of improvements including:
 - A new asynchronous API with full asyncio support.
 - Add support for :attr:`~pymongo.encryption.Algorithm.RANGE` and deprecate
   :attr:`~pymongo.encryption.Algorithm.RANGEPREVIEW`.
+- Added support for In-Use Encryption range queries with MongoDB 8.0.
+  Added :attr:`~pymongo.encryption.Algorithm.RANGE`.
+  ``trim_factor`` is not optional in :class:`~pymongo.encryption_options.RangeOpts`.
```

Comment:
``trim_factor`` is not optional -> ``sparsity`` and ``trim_factor`` are now optional

Reply:
Updated